src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -400,19 +400,57 @@ The main troubles you may have to face are the following ones:
 * The node seems to be well started but you cannot see it in the Resource Manager interface : in this case, make sure that the *port number* is the good one. Do not forget that the runtime port number is incremented from the initial ProActive Resource Manager port number. You can see exactly on which port your runtime has been started looking at the log file described above.
 
+== Available Network Protocols
+
+ProActive Workflows and Scheduling offers several communication protocols for
+the Scheduler and nodes to communicate. Each protocol provides different
+features: _speed_, _security_, _fast error detection_, _firewall_ or
+_NAT friendliness_.
+None of the protocols can offer all these features at the same time.
+Consequently, the selection should be made carefully. Below are introduced
+available network protocols.
+Configuration and properties are discussed in <<_network_properties>>.
+
+=== ProActive Network Protocol
+
+ProActive Network Protocol (PNP) is the general purpose communication protocol
+(`pnp://` scheme). Its performances are quite similar to the well known Java RMI
+protocol, but it is much more robust and network friendly. It requires only one
+TCP port per JVM and no shared registry. Besides, it enables fast network
+failure discovery and better scalability.
+PNP binds to a given TCP port at startup. All incoming communications use this
+TCP port. Deploying the Scheduler or a node with PNP requires to open one and
+only one incoming TCP port per machine.
+
+=== ProActive Network Protocol over SSL
+
+ProActive Network Protocol over SSL (PNPS) is the PNP protocol wrapped inside
+an SSL tunnel. The URI scheme used for the protocol is `pnps://`. It provides
+the same features as PNP plus ciphering and optionally authentication.
+Obviously, using SSL incurs some CPU overhead which implies that PNPS is slower
+than PNP.
+
+=== ProActive Message Routing
+
+ProActive Message Routing (PAMR) allows the deployment of the Scheduler and nodes
+on unfriendly networks (e.g. when firewalls are configured). Its associated URI
+scheme is `pamr://`. PAMR has the weakest expectations on how the network is
+configured. Unlike all the other communication protocols introduced previously,
+it has been designed to work when only outgoing TCP connections are available.
+
 == Installation on a Cluster with Firewall
 
-When the configuration of the network is unfriendly (ports closed, firewalls,...) the ProActive Scheduler allows
-you to connect nodes without significant changes in your network firewall configuration.
-It relies on the *PAMR* protocol (ProActive Message Routing Protocol) that has the weakest expectations on
-how the network is configured.
+When the configuration of the network is unfriendly (ports closed, firewalls,
```

Comment:
'unfiendly' is vague. You can say: When incoming connections are are not allowed (ports close, firewalls),